apps/expo-go/ios/Exponent/Versioned/Core/Api/Cognito/RNAWSCognito.h

```diff
@@ -1,16 +0,0 @@
-/**
- NOTE: the imports are slightly changed by hand for expo versioning.
- since cognito sdk does not change frequently, these changes are not included in `update-vendored-module` script. you should modify manually whenever upgrading the module.
- changes we did:
-   - replace imports from double-quote "" to bracket <> for xcode to find the correct versioning headers and clang modules.
- */
-
-#import <React/RCTBridgeModule.h>
-#import <React/RCTLog.h>
-#import <React/RCTUtils.h>
-
-#import <JKBigInteger/JKBigInteger.h>
-
-@interface RNAWSCognito : NSObject <RCTBridgeModule>
--(NSString*)getRandomBase64:(NSUInteger)byteLength;
-@end
```

Comment:
iirc this module is required for the vendored `amazon-cognito-identity-js` module (though it's not a truly native module). we still need it, please see #14416 for some reference

Reply:
They have deprecated that module. It now needs native code https://docs.amplify.aws/gen1/react-native/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#dropping-support-for-expo-go

Reply:
interesting that they also have a section for expo-go. cool it's a good time to remove cognito support, we should also follow up with:
- remove `amazon-cognito-identity-js` from `et uvm` list
- remove `JKBigInteger` for ios expo-go.
- remove same module in android expo-go